src/assets/translations/en/main.json

```
@@ -507,8 +507,8 @@
         "awaitingButtonPress": "Follow the steps on your KeepKey to view your Recovery Sentence. Write it down!"
       },
       "wiped": {
```

As this is no longer representing a wiped state, but rather a generic "KeepKey with factory settings connected", we should rename this key.

```suggestion
      "factoryState": {
```

or something similar.

---

Thanks for the tip! It lead me to another question -- I see lots of translation files, should I use smth like google-translate to fix them all, or better wait for some native speakers/translators?

---

That's a good question. As the key's value has changed, all translated strings for that key are now invalid.

We'd probably want to remove the key in the other languages, to be re-added by the globalization team, though I'm not sure we have a defined process.

@0xdef1cafe @firebomb1 is this the case?

---

> @0xdef1cafe @firebomb1 is this the case?

Indeed, there is no defined process yet, but I've mentioned this in a couple other issues/PR, if a string is changed/updated in English in a manner that requires translating them again you should nullify the value of this string in all the language files. This way they will appear to translators on the translating platform the next time they work on it.

Currently the platform (Gitlocalize) doesn't track "source" changes like these to "invalidate translations", so it is a manual process. I think we've discussed with @gomesalexandre at some point about this and he said it could be automated some way (tracking changes in en/main.json and nullifying the strings before they get merged, or at least add an additional "check" before the merge can happen)

---

Done, I removed keys in all translation files except the English one

---

Great context, thank you @firebomb1!

Cheers for the updated commit @Caranell.